### PR TITLE
Full repository review by Volodymyr Kozariz

### PR DESCRIPTION
**Pull request by odvova**

### CODE REVIEW

File: Suvorov.LNU.TwitterClone.Algorithms/PostRecomendations.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+        public PostRecomendations(PostService postService)
+        {
+            _postService = postService;
+        }
+
+        public async Task<List<Post>> GeneratePostRecomendations()
+        {
+            // Should be modifiyed when will add `Likes` to `Post`.
+            var sortedPosts = await _postService.SortByPostDate();
```
Comment (odvova) [inline Suvorov.LNU.TwitterClone.Algorithms/PostRecomendations.cs:18]: `SortByPostDate()` should be changed to a real recomendation algorithm

File: Suvorov.LNU.TwitterClone.Models/Frontend/CreateUserRequest.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        [Required(ErrorMessage = "Email is required.")]
+        [EmailAddress(ErrorMessage = "Enter valid email address.")]
+        public string? EmailAddress { get; set; }
+
+        [Required]
+        [DataType(DataType.Date)] 
+        public DateTime Birthday { get; set;}
+
+        [Required(ErrorMessage = "Password is required.")]
+        [RegularExpression("^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[a-zA-Z\\d]{8,}$",
```
Comment (odvova) [inline Suvorov.LNU.TwitterClone.Models/Frontend/CreateUserRequest.cs:27]: Add more error info, I entered `!` and still got the same message

**Review by odvova — COMMENTED** [review]
Made a full repository review

File: Suvorov.LNU.TwitterClone.Algorithms/PostRecomendations.cs (hunk 1 above)
Comment (olesuv) [reply Suvorov.LNU.TwitterClone.Algorithms/PostRecomendations.cs:18]: Yeah, will modify this when will add `Like` to `Post`

File: Suvorov.LNU.TwitterClone.Models/Frontend/CreateUserRequest.cs (hunk 2 above)
Comment (olesuv) [reply Suvorov.LNU.TwitterClone.Models/Frontend/CreateUserRequest.cs:27]: kk, will add more errors handling

**Pull request merged**